stable/yugaware/templates/configs.yaml

```
@@ -434,6 +434,16 @@ data:
           - source_labels: ["pod_name"]
             regex: "(.*)yb-(.*)"
             action: keep
+          # rename new name of the CPU metric to the old name and label
+          # ref: https://github.com/kubernetes/kube-state-metrics/blob/master/CHANGELOG.md#v200-alpha--2020-09-16
+          - source_labels: ["__name__", "unit"]
+            regex: "kube_pod_container_resource_requests;core"
+            target_label: "__name__"
+            replacement: "kube_pod_container_resource_requests_cpu_cores"
+          # Keep metrics for CPU, discard duplicate metrics
+          - source_labels: ["saved_name"]
```

```suggestion
          - source_labels: ["__name__"]
```
If we use saved_name, we don't get any metrics when using latest kube-state-metrics, because the saved_name is created at the first, and it holds the new name.